lib/rs/test/thrifts/Base_One.thrift

```diff
@@ -37,6 +37,9 @@ const list<double> CommonTemperatures = [300.0, 450.0]
 
 const double MealsPerDay = 2.5;
 
+const string DefaultRecipeName = "Something"
```

Comment:
This is incredibly minor, but, could you please choose some funny recipe names here? :)